react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/RNCustomerCenterModule.kt

```diff
@@ -119,11 +119,10 @@ internal class RNCustomerCenterModule(
                 purchaseIdentifier: String?
             ) {
                 val params = WritableNativeMap().apply {
-                    putString("option", action)
                     putString("actionId", customAction)
                     putString("purchaseIdentifier", purchaseIdentifier)
                 }
-                sendEvent("onManagementOptionSelected", params)
+                sendEvent("onCustomActionSelected", params)
```

Comment:
Ideally we would share these across platforms, but I guess that's the limitation of hybrid <-> native for now.